### PR TITLE
Added option to remove questions from output PDFs

### PR DESCRIPTION
**Pull request by markoprodanovic**
* Prompts user for whether to include questions in output PDFs [y/n]
* If y or Y, will output with questions -- will exclude questions if not
* saves this logic to a variable in settings.py

### CODE REVIEW

File: src/interface.py [hunk 1]
```diff
@@ -97,30 +97,40 @@ def get_user_inputs():
         has_question_bank = True
     else:
         has_question_bank = False
+    
+    include_questions_in_pdf = input('Would you like questions to appear in the output PDFs? [y/n]: ')
+
+    if include_questions_in_pdf == 'y' or include_questions_in_pdf == 'Y':
```
Comment (alisonmyers) [inline src/interface.py:103]: `if include_questions_in_pdf.upper() == 'Y'`

simplified logic?

**Review by alisonmyers — APPROVED** [review]
Approved with some suggestions - see notes in review.

Let's also make sure we add something in the README that describes why you would or wouldn't include question text.

File: src/interface.py (hunk 1 above)
Comment (markoprodanovic) [reply src/interface.py:103]: Nice, I like it!

Comment (markoprodanovic) [reply src/interface.py]: Implemented

File: README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@
 - **ID** (randomly-generated, anonymous student identifier)
 - **Course name** (as it appears on Canvas)
 - **Question text** (as it appears on Canvas)
+  > ⚠️ During execution you'll be given the option to exclude question text in the output pdfs. Question text may be useful to give context about which question is being answered. On the other hand, you may not want to include question text as the text will match across submissions to Turnitin. Ensure that you review the output to be sure it is suitable for your use case.
```
Comment (markoprodanovic) [inline README.md:26]: > Let's also make sure we add something in the README that describes why you would or wouldn't include question text.

Added to both README and Jupyter notebook

**Pull request merged**